circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 4.6.0
+    version: 6.7.0
```

Comment:
Is this advisable? Cos the circleCI build is failing currently and it's probably cos of this?